setup.cfg

```diff
@@ -1,14 +1,84 @@
+[metadata]
+name = sphinxcontrib-programoutput
+url = https://sphinxcontrib-programoutput.readthedocs.org/
+project_urls =
+  Bug Tracker = https://github.com/sphinx-contrib/sphinxcontrib-programoutput/issues
+  CI: GitHub = https://github.com/sphinx-contrib/sphinxcontrib-programoutput/actions?query=workflow:gh+branch:master+event:push
+  Documentation = https://sphinxcontrib-programoutput.readthedocs.org/
+  Source Code = https://github.com/sphinx-contrib/sphinxcontrib-programoutput
+description = Sphinx extension to include program output
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Sebastian Wiesner
+author_email = lunaryorn@gmail.com
+maintainer = Jason Madden
+maintainer_email = jason@nextthought.com
+license = BSD
+license_file = LICENSE
```

Comment:
`license_file` is deprecated in favor of `license_files` (plural) as of `wheel==0.32`. Also, if you omit it, `wheel` will attempt to include all the files that have common license file names: https://wheel.rtfd.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file.